### PR TITLE
Added a new question about bubbling vs capturing.

### PR DESCRIPTION
**Pull request by saikota**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ This file contains a number of front-end interview questions that can be used wh
 * How is responsive design different from adaptive design?
 * Have you ever worked with retina graphics? If so, when and what techniques did you use?
 * Is there any reason you'd want to use `translate()` instead of *absolute positioning*, or vice-versa? And why?
+* How to create an Up/Down arrow using CSS Styles.
```
Comment (darcyclarke) [inline README.md]: I'm not sure what you're looking to find out about a candidate's knowledge by answering this kind of question. Could you explain a bit more about your thinking here?

**Comment by phanan** [comment]
Is this answer valid? `.elem::after{ content: '← ↑ → ↓ ↔ ↕ ⇪ ↹ ⬈ ↘ ⟾ ↶'; }` ? :smile:

File: README.md (hunk 1 above)
Comment (saikota) [inline README.md]: i was asked this question in an interview. the expected solution involved setting the height and width to 0 and then use the border property to create the arrow effect.
http://codepen.io/alexandredees/pen/xIKow

Comment (darcyclarke) [inline README.md]: I know the technique you're referencing but there are a lot of ways to create shapes in HTML and CSS (not just using borders). That said, SVG seems to be the preferred method over custom fonts, pseudo elements or CSS hacks; It's more flexible for animation and was, essentially, made for this kind of thing (ie. vector graphics). 

How about, instead, we ask something like:
> How would you create a non-[rectilinear](http://en.wikipedia.org/wiki/Rectilinear_polygon) shape for a website? Are there any advantages or disadvantages to your approach?

**Rectilinear** is probably an unfamiliar term for many people but the idea is that you want someone to think *outside the box* (ie. shapes not composed of squares or rectangles). I'm open to other wording if anyone else wants to take a crack at it.

File: README.md [hunk 2]
```diff
@@ -118,6 +119,7 @@ This file contains a number of front-end interview questions that can be used wh
   * If so, what libraries have you used?
 * Explain "hoisting".
 * Describe event bubbling.
+* What is the difference between event bubbling and event capturing.
```
Comment (darcyclarke) [inline README.md]: We already have a question about event bubbling (which is just above this one). Maybe just focus on event capturing. 

> Describe event capturing.

**Comment by saikota** [comment]
will separate the questions and resubmit.

**Pull request closed without merging**